### PR TITLE
Add tv ad-example styles to SCSS

### LINKED ISSUE #3373: CSS Styles for TV ad examples
### Summary

**What we're after:**
We need to move CSS in the advertising reporting examples to SCSS files, instead of having it in the html block. It could probably go in `components/_responsive-object.scss` 

Draft page:
https://fec-prod-proxy.app.cloud.gov/admin/pages/10616/edit/preview/

### Related issues
https://github.com/fecgov/fec-cms/issues/3179

**Action Items**
- [x] Remove CSS from any html blocks using the TV ad example iframe background image.
- [x] Add or edit classnames where  necessary html block in Wagtail and in `components/_responsive-object.scss`  for semantic clarity

### PR DESCRIPTION
**Pull request by johnnyporkchops**
## Summary (required)
- Remove CSS from any Wagtail html blocks using the TV ad example iframe pattern
- Add or edit classnames where necessary in html structure  in Wagtail 
- Add to `_responsive-object.scss` and include example html structure there

Resolves #3373

**After deployment:**
Make changes to these three live pages to reflect the simplified structure (right under 'How to test' below). Save as draft to be published after PR is deployed to production as part of a release(existing pages wont break upon deploying this, we just need to change them after):

- [x] Make `after release` ticket to ensure the below items are completed after the  release. **See https://github.com/fecgov/fec-cms/issues/3840**

- [ ] **Edit:** https://fec-prod-proxy.app.cloud.gov/admin/pages/10811/edit/
  **Live page:** https://www.fec.gov/help-candidates-and-committees/making-disbursements-political-party/party-committee-independent-expenditure-tv-disclaimer-ad/

- [ ]  **Edit:** https://fec-prod-proxy.app.cloud.gov/admin/pages/10616//edit/
  **Live page:** https://www.fec.gov/help-candidates-and-committees/making-disbursements/advertising/candidate-television-ad-example/

 - [x] There is a new page  being created by @djgarr,  but will likely not be published before  release and he has been updated on the new structure for making these items in html blocks.

 

## Impacted areas of the application
modified:   fec/static/scss/components/_responsive-object.scss
Live Wagtail pages using the TV ad example pattern. (listed above)

## How to test
-  checkout and run: `feature/css-styles-tv-examples`
- `npm run-build sass`
- Edit http://127.0.0.1:8000/admin/pages/10811/edit/
- Replace the html block with just this:
```
<div class="video-frame">
   <div class="iframe-container">
    <iframe src="https://www.youtube.com/embed/REOm_O2VJcc" rel=0 frameborder="0" allowfullscreen></iframe>
   </div>
</div>
```
- Preview or publish and confirm that it still renders just like the live version:
https://www.fec.gov/help-candidates-and-committees/making-disbursements/advertising/candidate-television-ad-example/
- Test responsivity  of the patterm

NOTE: If you get error on previewing on your local dev servr, you may need to change `Wagtail >Settings > Sites > Site > Hostname` to localhost. Or you can just publish locally and test that way.
____

### CODE REVIEW

**Comment by johnnyporkchops** [comment]
For third  checklist item above, the new TV example page will likely not go live before this PR is merged, so we only need to change the two pages listed above.

File: fec/fec/static/scss/components/_responsive-object.scss [hunk 1]
```diff
@@ -7,3 +7,44 @@
...
+  background-size: cover;
+  margin-bottom: u(2rem);
+
+  .iframe-container {
+    overflow: hidden;
+    padding-top: 56.25%;
+	position: relative;
+
+      iframe {
+	    height: 62%;
```
Comment (rfultz) [inline fec/fec/static/scss/components/_responsive-object.scss:33]: Have we checked this in IE? Last I checked it wanted videos to have heights that were actual numbers and not just percentages, but that's been a while

File: fec/fec/static/scss/components/_responsive-object.scss [hunk 2]
```diff
@@ -7,3 +7,44 @@
...
+		border-bottom: 27px solid #333;
+	  }
+   }
+}
+
+ 
+
+
+
+
```
Comment (rfultz) [inline fec/fec/static/scss/components/_responsive-object.scss:50]: The linter will want two-space indents and exactly one blank line at the bottom.

Should we use quotes inside the url()? I go back & forth

Comment (johnnyporkchops) [reply fec/fec/static/scss/components/_responsive-object.scss:50]: I go back and forth too. We should decide on one or the other

File: fec/fec/static/scss/components/_responsive-object.scss (hunk 1 above)
Comment (johnnyporkchops) [reply fec/fec/static/scss/components/_responsive-object.scss:33]: @robert, this works in IE. I am basing it loosely on this technique. On a side note,  Wagtail used to add to this CSS to  draftail video embed as well, but I think I read they removed it and leave it up to us so do in current versions.
https://alistapart.com/article/creating-intrinsic-ratios-for-video/#section4

**Comment by johnnyporkchops** [comment]
@robert, I made the spacing changes. If you are good with it, you can merge

**Pull request merged**